src/ln/functional_tests.rs

```diff
@@ -5825,6 +5697,240 @@ fn test_dynamic_spendable_outputs_local_htlc_success_tx() {
 	check_spends!(spend_txn[1], node_txn[2].clone());
 }
 
+fn do_test_fail_backwards_unrevoked_remote_announce(deliver_last_raa: bool, announce_latest: bool) {
+	// Test that we fail backwards the full set of HTLCs we need to when remote announces its
```

Comment:
nit: "announces" intents that remote tell us its latest commitment transaction via some offchain messages, "broadcast" wouldn't be more meaningful there ?